Project qualifications.md

```diff
@@ -0,0 +1,91 @@
+# Project Qualifications
+
+This document outlines what projects qualify as entries.
+The goal of this list, in short, is:
+
+**Provide a list of awesome open source projects, that**
+**try to improve the world in some fashion, to engineers**
+**who want to altruistically participate in those.**
+
+As such, every project will be evaluated based on their
+fitness for this goal.
+A project that is valid will then be assigned a rank
+based on how friendly it is for engineers to participate
+in.
+
+## Which projects are allowed in the first place?
+
+In order to provide a sensible list, every project has to
+fulfill the following conditions in order to be added:
+
+### 1. The project must accept contributions
+
+This is rather obvious:
+If it isn't accepting code from other users, there is no
```

Comment:
If it's obvious, do we really need to write about it?

Reply:
Well, my thought process was to add it, since it is a requirement for the project – an obvious one, true, but it still might be nice to write it down in a document?
If you feel like it's useless, I don't mind removing it.